src/hooks.server.ts

```
@@ -26,6 +26,24 @@
 		if (!event.locals.backendUrl || !event.locals.apiKey) {
 			throw redirect(307, '/connect');
 		}
+
+		let hasConnection;
+
+		try {
+			await fetch(event.locals.backendUrl + '/api/v1/health')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)